src/Orleans.Runtime/Messaging/OutboundMessageQueue.cs

```diff
@@ -82,34 +84,23 @@ public void SendMessage(Message msg)
             {
                 if (stopped)
                 {
-                    logger.Info(ErrorCode.Runtime_Error_100115, "Message was queued for sending after outbound queue was stopped: {0}", msg);
+                    logger.LogInformation((int)ErrorCode.Runtime_Error_100115, "Message was queued for sending after outbound queue was stopped: {Message}", msg);
+                    messageCenter.SendRejection(msg, Message.RejectionTypes.Unrecoverable, "Message was queued for sending after outbound queue was stopped");
```

Comment:
This is the important line (and 52, which is identical). The other changes are cleanliness changes.